### PR TITLE
RWA-1522 - Remove task configuration code

### PR DESCRIPTION
**Pull request by sullivangareth**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RWA-1522

### Change description ###

Remove task configuration code

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by burale — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/camunda/bpm/config/EventHandlerConfiguration.java [hunk 1]
```diff
@@ -2,51 +2,17 @@
...
-            LOG.info(EVENT_RECEIVED_LOGGER_MESSAGE, "CREATE", delegateTask.getId());
-            /*
-            Uses DelegateTask as it is a mutable object
-            Call wa-task-configuration to retrieve configuration for a tasks.
-            The reason it is done in this way is because the tasks does not yet exist in the database
-            when this event is triggered
-            */
-
-            taskConfigurationService.configureTask(delegateTask);
-            delegateTask.setVariableLocal(CFT_TASK_STATE_LOCAL_VARIABLE_NAME, "unconfigured");
```
Comment (raghera) [inline src/main/java/uk/gov/hmcts/reform/camunda/bpm/config/EventHandlerConfiguration.java:39]: we will want to keep this line.  Discussed with @burale so don't think we can remove this handler completely.

**Review by raghera — CHANGES_REQUESTED** [review]
The ticket this depends on has a problem with it.  On review we can't remove this method completely since we still need one line which is essential for Task creation.

File: src/main/java/uk/gov/hmcts/reform/camunda/bpm/config/EventHandlerConfiguration.java (hunk 1 above)
Comment (rafalkalita) [reply src/main/java/uk/gov/hmcts/reform/camunda/bpm/config/EventHandlerConfiguration.java]: Updated the PR. Left the line untouched and updated tests.

**Review by raghera — APPROVED** [review]

**Pull request merged**